server/models/User.js

```diff
@@ -52,7 +52,7 @@ userSchema.methods.isCorrectPassword = async function (password) {
 
 const User = model("User", userSchema);
 
-module.exports = User;
+module.exports = userSchema;
```

Comment:
I think we'll still want to use the User model here instead of the schema.

```suggestion
module.exports = User;
```